### PR TITLE
fixed usertags (wrong model)

### LINKED ISSUE #329: Tagged users in pictures
Hi all, I'm triying to get the users tagged in a picture from a InstagramFeedItem using the method getUsertags(). Sometimes the size of the returned array is zero (that I supose means nobody tagged) and sometimes the size is 1. In the last case, if I check the element of the array I get an object like this: `InstagramFeedUserTag(super=org.brunocvcunha.instagram4j.requests.payload.InstagramFeedUserTag@5cc69cfe, user=null, position=[0.0, 0.0], start_time_in_video_in_sec=0.0, duration_in_video_in_sec=0.0)`

If I check the picture in Instagram I see there is one or more people tagged in the picture. Am I doing something wrong or it's a bug?

In case anybody have a working code about this I'll be very pleased if you share an example.

Thanks and congratulations for the amazing work done @brunocvcunha

### PR DESCRIPTION
**Pull request by jemand771**
Closes #329

### CODE REVIEW

**Comment by HectorHumanes** [comment]
Hi @jemand771, can you solve the conflicts in order to get this fix released in further versions? This will be very much appreciated :)

**Comment by jemand771** [comment]
oh, sorry! I totally thought only admins could do that! 😅 
I hope the code still works as its quite old now already

**Comment by brunocvcunha** [comment]
Thanks!

**Comment by gunther82** [comment]
How can I get the version with this update? The version 1.12 updated by Maven still contains usertags as List<InstagraFeedUserTag> in InstagramFeedItem.
Thanks

**Comment by jemand771** [comment]
@gunther82 yup, you'll need to wait for the next release
Merged just means that the feature will certainly be in the next update :)

**Comment by gunther82** [comment]
Ah ok thank you, I'll wait for the next release :)

File: src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java [hunk 1]
```diff
@@ -46,8 +46,8 @@
     public Map<String, Object> attribution;
     public List<ImageMeta> video_versions;
     public ImageVersions image_versions2;
-    public List<InstagramFeedUserTag> usertags;
-    public FeedItemLocation location;
+    public InstagramUserTagsContainer usertags;
+    public Map<String, Object> location;
```
Comment (tayfunyasar) [inline src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java]: Why did you removed FeedItemLocation obj?

Comment (jemand771) [reply src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java]: was moved to InstagramUserTagsContainer. The model structure has changed

Comment (tayfunyasar) [reply src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java]: Thanks but what about FeedItemLocation? Was it necesary to replace "location" variable FeedItemLocation to Map?

Comment (jemand771) [reply src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java]: just realized i misread your first message, sorry. i honestly don't know why i did that. probably had a reason to do so, but i don't know what it was anymore..

**Pull request merged**